MetadataMigration/src/main/java/com/rfs/MetadataMigration.java

```diff
@@ -78,6 +84,13 @@ public static class Args {
             + " This can be useful for migrating to targets which use zonal deployments and require additional replicas to meet zone requirements.  Default: 0")
             , required = false)
         public int minNumberOfReplicas = 0;
+
+        @Parameter(required = false,
+                names = {"--otelCollectorEndpoint"},
+                arity = 1,
+                description = "Endpoint (host:port) for the OpenTelemetry Collector to which metrics logs should be" +
```

Comment:
do you mean scheme:host:port?

Reply:
We pass the argument onto [this call](https://javadoc.io/doc/io.opentelemetry/opentelemetry-exporter-otlp/0.12.0/io/opentelemetry/exporter/otlp/OtlpGrpcSpanExporter.Builder.html#setEndpoint(java.lang.String)) in Otel.  It simply says "Sets the OTLP endpoint to connect to. Optional, defaults to "localhost:4317".

Since the endpoint runs OTLP, specifying http or https is probably incorrect.  We should remove them from our docs and our deployment code (& double check).